languages/elixir/exercises/concept/basic-strings/.docs/after.md

```diff
@@ -0,0 +1,111 @@
+[Strings][getting-started-strings] in Elixir are delimited by double quotes, and they are encoded in UTF-8:
+
+```elixir
+iex> "Hi! ¡Hola! Cześć! Привет!"
+"Hi! ¡Hola! Cześć! Привет!"
+```
+
+Strings can be concatenated using the [`<>/2`][kernel-concat] operator:
+
+```elixir
+iex> "Welcome to" <> " " <> "New York"
+"Welcome to New York"
+```
+
+Strings in Elixir support [interpolation][string-interpolation] using the `#{}` syntax:
+
+```elixir
+iex> "6 * 7 = #{6 * 7}"
+"6 * 7 = 42"
+```
+
+Any Elixir expression is valid inside the interpolation. If a string is given, the string is interpolated as is. If any other value is given, Elixir will attempt to convert it to a string.
+
+```elixir
+iex> "Elixir can convert booleans to strings: #{true}"
+"Elixir can convert booleans to strings: true"
+
+iex> "And #{["lists", ", ", "too!"]}"
+"And lists, too!"
+
+iex> "But not functions: #{fn x -> x end}"
```

Comment:
Will students know about this anonymous function syntax at this point @neenjaw? If so, this is fine. If not, I think I would remove it.

Reply:
There already is a `anonymous-functions` concept exercise that requires only the `basics` concept, but I don't know what needs to happen to _ensure_ that it's taught before strings.

I wanted to present the `"protocol String.Chars not implemented for"` error here explaining that it means you can't put certain values in a string because it's a cryptic error for beginners (protocols...) but likely to happen. I couldn't come up with a simpler data type that would cause it though... could also be a tuple or a map, but those concepts already have concept exercises here that have higher prerequisites than anonymous functions.